src/components/schedule-visualizer/index.tsx

```diff
@@ -304,11 +310,32 @@ export default function ScheduleVisualizer(props: ScheduleVisualizerProps): Reac
                     conflicts={curMapConflicts}
                     colorManager={colorManager}
                     conflictRobotNames={conflictRobotNames}
+                    overridePathColor={undefined}
```

Comment:
You can remove this `overridePathColor={undefined}`. If you don't send the parameter the child will receive undefined. 

Ditto for all the components below

Reply:
removed unneeded parameter a895dae